src/FieldWarning/Assets/Model/Game/Loading.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
```

Comment:
Missing license

Reply:
Thanks for the review!

Reply:
The new files are still missing the license header (you can copy it from any existing code file in the project).

Reply:
apologies... not sure what happened there, thought I put the header up everywhere.